core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/controller/ControllerAsyncRPCHandlerInitializer.scala

```diff
@@ -51,6 +51,13 @@ class ControllerAsyncRPCHandlerInitializer(
 
   var statusUpdateAskHandle: Option[Cancellable] = None
 
+  var monitoringHandle: Option[Cancellable] = None
```

Comment:
can we move all reshape-related variables to another class?

Reply:
done